pkg/Library/NodeGraph/ConnectionUpdator.js

```diff
@@ -28,19 +27,23 @@ update({pipeline, nodeTypes, candidates}, state) {
   }
 },
 
+inputsChanged({pipeline, candidates}, state) {
+  return pipeline && 
```

Comment:
I would not have noticed this probably except by doing code review, but the contract of `pipelineChanged` is different between this module and `CandidateFinder.js`. It's a micro-nit. :)

Reply:
good catch (thanks for the NodesConnector refactoring and breaking it into 3 separate particles, in 2 of these, the pipelineChanged implementation could be significantly simplified - work only needed, if switching to another pipeline, or node(s) added/removed)